internal/cmd/connect/connect.go

```diff
@@ -207,7 +213,9 @@ argument:
 	cmd.PersistentFlags().StringVar(&flags.execCommandEnvURL, "execute-env-url", "DATABASE_URL",
 		"Environment variable name that contains the exposed Database URL.")
 	cmd.PersistentFlags().StringVar(&flags.role, "role",
-		"admin", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
+		"", "Role defines the access level, allowed values are: reader, writer, readwriter, admin. Defaults to 'reader' for replica passwords, otherwise defaults to 'admin'.")
```

Comment:
Since we took out the default `admin` here, should we check for empty string above and set to admin if not replica? (to preserve old functionality)

Reply:
@notfelineit Yeah, we do that by default already. The role is set to `admin` by default in the code and if the replica flag is enabled, we override it to be `reader`.

Reply:
Wow. I'm blind cause I totally missed that!